color/Cargo.toml

```diff
@@ -12,10 +12,13 @@ edition = "2021"
 path = "lib.rs"
 
 [dependencies]
-cssparser = { path = ".." }
-serde = { version = "1.0", features = ["derive"], optional = true }
+cssparser = { path = "..", default-features = false }
+serde = { version = "1.0", default-features = false, features = ["derive"], optional = true }
+libm = "0.2.8"
```

Comment:
Ah, I missed this. It seems this dependency should be optional, and only enabled if `std` is disabled?